### PR TITLE
Fix cxxmodules build for Mac OS X

### PR DESCRIPTION
**Pull request by oshadura**

### CODE REVIEW

File: roofit/histfactory/config/prepareHistFactory [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
 # HistFactory workplace setup script
-
-ROOTETCDIR=`root -b -q -l -n -e "std::cout << TROOT::GetEtcDir() << std::endl;" | tail -1`
+# There is no root binary on Mac OSX (root depends on x11), switching to root.exe
+ROOTETCDIR=`root.exe -b -q -l -n -e "std::cout << TROOT::GetEtcDir() << std::endl;" | tail -1`
 echo "Using etcdir "$ROOTETCDIR
-ROOTTUTDIR=`root -b -q -l -n -e "std::cout << TROOT::GetTutorialDir() << std::endl;" | tail -1`
-echo "Using tutorials dir" $ROOTTUTDIR 
+ROOTTUTDIR=`root.exe -b -q -l -n -e "std::cout << TROOT::GetTutorialDir() << std::endl;" | tail -1`
+echo "Using tutorials dir" $ROOTTUTDIR
```
Comment (amadio) [inline roofit/histfactory/config/prepareHistFactory]: There is now `root-config --tutdir`.

Comment (hageboeck) [reply roofit/histfactory/config/prepareHistFactory]: +1!

Comment (oshadura) [reply roofit/histfactory/config/prepareHistFactory]: > There is now `root-config --tutdir`.

Done, thanks!

**Comment by pcanal** [comment]
@phsft-bot build with flags -DCTEST_TEST_EXCLUDE_NONE=On

**Pull request merged**